Tests/Runner/BaseTableBackendTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Keboola\DockerBundle\Tests\Runner;
+
+use Keboola\DockerBundle\Tests\BaseRunnerTest;
+
+abstract class BaseTableBackendTest extends BaseRunnerTest
+{
+    use BackendAssertsTrait;
+
+    abstract public static function expectedDefaultTableBackend(): string;
+
+    public function setUp(): void
+    {
+        if (!constant(sprintf('RUN_%s_TESTS', mb_strtoupper($this::expectedDefaultTableBackend())))) {
```

Comment:
constant -> getenv

Reply:
Ono to bylo pres konstanty, ktere jsou definovane v bootstrapu. Tim ale ze jsem to prehodil na condition v pipelines tak se tohle vyhodilo